### DISCUSSION TITLE
GCP Console Support ETA

### DISCUSSION THREAD

**Discussion by MrBlaise** [gravitational/teleport#12866] [Q&A] [answered]
Hi!

I saw that only AWS console access is supported. Is there any ETA on GCP console support?

Thanks a lot!

**Comment by Valien** [comment, marked as answer]
No public ETA as of now. You can check this page for any potential updates - https://goteleport.com/docs/preview/upcoming-releases/?scope=enterprise

**Reply by MrBlaise** [reply]
I see, thanks!
